documentation/overview.md

```diff
@@ -264,29 +262,26 @@ The entity definition is retrieved from
 ```scala
 df.write.format("com.microsoft.cdm")
  .option("storage", "mystorage.dfs.core.windows.net")
- .option("container", "cdmdata")
- .option("manifest", "/contacts/root.manifest.cdm.json")
+ .option("manifestPath", "cdmdata/contacts/root.manifest.cdm.json")
  .option("entity", "Person")
- .option("entityDefinitionContainer", "cdmmodels")
- .option("entityDefinitionModelRoot", "core")
- .option("entityDefinition", "/Contacts/Person.cdm.json/Person")
+ .option("entityDefinitionModelRoot", "cdmmodels/core")
+ .option("entityDefinitionPath", "/Contacts/Person.cdm.json/Person")
  .mode(SaveMode.Overwrite)
  .save()
 ```
 #### Explicit Write - using an entity defined in the CDM GitHub
 
-This code writes the dataframe _df_ to a CDM folder with the manifest at [https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json](https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json) and a sub-manifest containing the TeamMembership entity, created in a TeamMembership subdirectory. TeamMembership data is written as CSV files (by default) that overwrite any existing data files. The entity definition is retrieved from the CDM GitHub repo, at:
-[https://github.com/microsoft/CDM/tree/master/schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership](https://github.com/microsoft/CDM/tree/master/schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership)
+This code writes the dataframe _df_ to a CDM folder with the manifest at [https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json](https://mystorage.dfs,core.windows.net/cdmdata/Teams/root.manifest.cdm.json) and a sub-manifest containing the TeamMembership entity, created in a TeamMembership subdirectory. TeamMembership data is written as CSV files (by default) that overwrite any existing data files. The entity definition is retrieved from the CDM CDN, at:
+[https://cdm-schema.microsoft.com/logical//schemaDocuments/core/applicationCommon/TeamMembership.cdm.json/TeamMembership](https://cdm-schema.microsoft.com/logical//core/applicationCommon/TeamMembership.cdm.json/TeamMembership)
```

Comment:
is this URL correct?

Reply:
Nope - fixed both parts, incl. removing dangling reference to schema documents